SEC/Equities/EquitiesExampleIndividuals.rdf

```diff
@@ -74,11 +82,126 @@
+		<fibo-sec-sec-iss:hasInstrumentDescription>SH CAP SH NV</fibo-sec-sec-iss:hasInstrumentDescription>
+		<fibo-sec-sec-iss:hasIssuerShortName>ALPHABET INC</fibo-sec-sec-iss:hasIssuerShortName>
+	</owl:NamedIndividual>
+	
+	<owl:NamedIndividual rdf:about="&fibo-sec-eq-eqind;AlphabetIncEquityIssuer">
+		<rdf:type rdf:resource="&fibo-fbc-fi-fi;Issuer"/>
+		<rdfs:label>Alphabet Inc. equity issuer</rdfs:label>
+		<skos:definition>individual representing the Alphabet Inc. functional entity that is an issuer of stock</skos:definition>
+		<fibo-fnd-rel-rel:hasIdentity rdf:resource="&fibo-be-le-usee;AlphabetInc-US-CA"/>
+		<fibo-sec-sec-iss:hasIssuerShortName>ALPHABET INC</fibo-sec-sec-iss:hasIssuerShortName>
+	</owl:NamedIndividual>
+	
+	<owl:NamedIndividual rdf:about="&fibo-sec-eq-eqind;AppleIncCommonStock">
+		<rdf:type rdf:resource="&fibo-sec-eq-eq;CommonShare"/>
+		<rdfs:label>Apple Inc. common stock</rdfs:label>
+		<skos:definition>common share that is a share in Apple Inc.</skos:definition>
+		<fibo-fbc-fi-fi:isLegallyRecordedIn rdf:resource="&fibo-be-ge-usj;StateOfCaliforniaJurisdiction"/>
+		<fibo-fbc-fi-fi:isNegotiable rdf:datatype="&xsd;boolean">true</fibo-fbc-fi-fi:isNegotiable>
+		<fibo-fnd-rel-rel:isIssuedBy rdf:resource="&fibo-sec-eq-eqind;AppleIncEquityIssuer"/>
+		<fibo-sec-sec-lst:hasHomeExchange rdf:resource="&fibo-fbc-fct-mkti;Exchange-XNAS"/>
+	</owl:NamedIndividual>
+	
+	<owl:NamedIndividual rdf:about="&fibo-sec-eq-eqind;AppleIncCommonStockFinancialInstrumentShortName">
+		<rdf:type rdf:resource="&fibo-sec-sec-iss;FinancialInstrumentShortName"/>
+		<rdfs:label>Apple Inc. common stock financial instrument short name</rdfs:label>
+		<skos:definition>individual representing the Apple Inc. common stock financial instrument short name (FISN)</skos:definition>
+		<fibo-fnd-aap-agt:identifies rdf:resource="&fibo-sec-eq-eqind;AppleIncCommonStock"/>
+		<fibo-sec-sec-iss:hasInstrumentDescription>SH SH</fibo-sec-sec-iss:hasInstrumentDescription>
+		<fibo-sec-sec-iss:hasIssuerShortName>APPLE INC</fibo-sec-sec-iss:hasIssuerShortName>
+	</owl:NamedIndividual>
+	
+	<owl:NamedIndividual rdf:about="&fibo-sec-eq-eqind;AppleIncEquityIssuer">
+		<rdf:type rdf:resource="&fibo-fbc-fi-fi;Issuer"/>
+		<rdfs:label>Apple Inc. common stock issuer</rdfs:label>
+		<skos:definition>individual representing the Apple Inc. functional entity that is an issuer of common stock</skos:definition>
+		<fibo-fnd-rel-rel:hasIdentity rdf:resource="&fibo-be-le-usee;AppleInc-US-CA"/>
+		<fibo-sec-sec-iss:hasIssuerShortName>APPLE INC</fibo-sec-sec-iss:hasIssuerShortName>
+	</owl:NamedIndividual>
+	
+	<owl:NamedIndividual rdf:about="&fibo-sec-eq-eqind;BBG000B9Y5X2">
+		<rdf:type rdf:resource="&fibo-sec-sec-idind;FinancialInstrumentGlobalIdentifier"/>
+		<rdfs:label>BBG000B9Y5X2</rdfs:label>
+		<skos:definition>Apple Inc. share class Financial Instrument Global Identifier (FIGI) listed in the Nasdaq (NASDAQ/NGS)</skos:definition>
```

Comment:
Use of "share class" here is quite misleading since it's not the FIGI of the share class. We should separately model that (as an identifier for the AppleIncCommonStock)

Reply:
We already have both, so perhaps the definition is misleading, but the individuals are properly modeled and reference the correct concept.

Reply:
Fixed - eliminated share class from those definitions.